app/src/main/java/org/oppia/app/player/state/itemviewmodel/SelectionContentViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package org.oppia.app.player.state.itemviewmodel
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+/** [ViewModel] for MultipleChoiceInput values or ItemSelectionInput values. */
+@FragmentScope
+class SelectionContentViewModel @Inject constructor() : ViewModel() {
+  var htmlContent: String =""
```

Comment:
Please remove unused string. This class seems not in use, if so please do remove.

Reply:
Removed unused class.